### DISCUSSION TITLE
Copilot for education

### DISCUSSION THREAD

**Discussion by krolchonok** [community/community#43833] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Good afternoon, there was a mention in the description of github copilot that access for students is free, but now I can't access it, although I have student status

**Comment by alvarosdev** [comment]
Same problem here :( 


```
> Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended.
> You are eligible to use GitHub Copilot for free.
```

**Comment by Cwarcup** [comment]
Check out #43840 or #43841. Looks like the issue persists.

**Comment by UmbraCi** [comment]
same issue

**Comment by WINDY-WINDWARD** [comment]
the issue came back for me even after getting a mail that says my access has been reinstated until February 8th.
its the same invalid token issue with error code 403

**Reply by petesmithofficial** [reply]
same.

**Comment by exposir** [comment]
same issue

**Comment by FishZe** [comment]
I have the same problem.

**Comment by Akash1134** [comment, marked as answer]
Hi there,

Some details and updates on this issue:

This week, we've had to take down some of our Copilot resources for maintenance. While we did our best to source extra capacity elsewhere, we've been forced to limit some free users for the period of time where one of our busiest models is out of commission.

We are also aware that some students are still experiencing issues with redeeming and accessing Copilot and that the team is currently working to resolve this. We'll update the community when we have more information on that incident in a separate discussion.

We expect that this will cease to impact your use of Copilot after the 16th of January. In the meantime, you will have less frequent suggestions, but you should still be getting suggestions - just with rate limits in between. You may see these errors specifically if you look in the output pane or logs for your IDE.

Thanks for your patience and apologies for the inconvenience caused.

**Reply by krolchonok** [reply]
Thanks for the explanation. Copilot is very useful anyway, thanks for supporting it

**Reply by doblem2000** [reply]
hello and thank you for your reply, I wanted to ask if there were still any problems, as being a member of GitHub Student I am not able to access GitHub Copilot
![1](https://user-images.githubusercontent.com/91904787/214262863-2b098888-914d-4b4c-8b2f-4fb8912a14f2.jpg)

**Reply by MZ-Sun** [reply]
Hi, same issue here. PLZ PLZ PLZ GRANT ME ACCESS TO COPILOT AGAIN. DESPERATELY WAITING.

**Comment by MZ-Sun** [comment]
Hi anyupdate?

**Comment by MustaphaHa** [comment]
Hello, my application for copilot for students was approved yesterday. How and when can i start using it?

**Comment by RONNCC** [comment]
Hey is there any update here?
